src/FlipCounter.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+     x:"000000"
```

Comment:
It is better to store the value as number (because you have add function and it will be easier to work in this way), and convert it to string inside render